src/Common/Core/Test/Fixtures/CoreShellProviderFixture.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition.Hosting;
+using System.Threading.Tasks;
+using Microsoft.Common.Core.Extensions;
+using Microsoft.Common.Core.Services;
+using Microsoft.Common.Core.Shell;
+using Microsoft.Common.Core.Test.Fakes.Shell;
+using Microsoft.UnitTests.Core.Shell;
+using Microsoft.UnitTests.Core.XUnit;
+using Microsoft.UnitTests.Core.XUnit.MethodFixtures;
+using Xunit.Sdk;
+
+namespace Microsoft.Common.Core.Test.Fixtures {
+    public abstract class CoreShellProviderFixture : MethodFixtureBase, ICoreShellProvider {
```

Comment:
This is the change that I don't understand. `IExportProvider` isn't bound to the MEF, it is pretty much the same `IServiceProvider` that supports some of the MEF specifics.

Since you don't need `T GetExportedValue<T>(string metadataKey, params object[] metadataValues) where T : class;` and `IEnumerable<Lazy<T, TMetadataView>> GetExports<T, TMetadataView>() where T : class;` anymore, you can rename it into `ITestServices` with the only method `T Get<T>()`, without any other changes. Many of the tests (72 out of 106 files) that use `IExportProvider` do not require `ICoreShell`